test-app/tests/integration/components/ember-flatpickr-test.js

```diff
@@ -456,6 +457,35 @@ module('Integration | Component | ember flatpickr', function (hooks) {
     );
   });
 
+  test('locale as string works correctly', async function (assert) {
+    assert.expect(1);
+
+    this.set('dateValue', '2080-12-01T16:16:22.585Z');
+    this.set('maxDate', '2080-12-31T16:16:22.585Z');
+    this.set('minDate', '2080-12-01T16:16:22.585Z');
+
+    this.set('locale', 'fr');
+
+    await render(hbs`<EmberFlatpickr
+      @date={{this.dateValue}}
+      @locale={{this.locale}}
+      @maxDate={{this.maxDate}}
+      @minDate={{this.minDate}}
+      @onChange={{null}}
+      placeholder="Pick date"
+    />`);
+
+    await waitFor('.flatpickr-current-month .flatpickr-monthDropdown-month');
```

Comment:
Was forced to add a waitFor. The setupFlapickr function must be asynchronous for lazy loading. It might have an impact on existing apps. Maybe should i use https://github.com/emberjs/ember-test-waiters ?

Reply:
edit: used @ember/test-waiters